scripts/translation2js.yaml

```diff
@@ -1,2 +0,0 @@
-langs: [de,fr,en,it,rm]
-emptyFilename: empty.json
```

Comment:
Currently this is not. I should add it. However, I see the point of the `empty.json` (not to include translations that are not ready if I'm right), I fail to see the utility of `langs`: langs are activated by topics so even if the translation for a lang is not ready in your spreadsheet it won't be activated on the topic.

Should I still support both?